tests/FixtureBuilder/Fixture.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace Tuf\Tests\FixtureBuilder;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * This is our in-house TUF fixture builder.
```

Comment:
I don't think we need to specify "in-house" because anybody new to the codebase would have to have the context that we use to use the Python TUF for this to make sense.